### ISSUE TITLE
org.apache.http.conn.util.InetAddressUtils not available in Android API level 23

### ISSUE THREAD

**Issue by tkem** [vstirbu/ZeroConf#18]
Compiling with Android API level 23 yields this error:

```
platforms/android/src/com/triggertrap/ZeroConf.java:31: error: package org.apache.http.conn.util does not exist
import org.apache.http.conn.util.InetAddressUtils;
```

Adding this to my project's `platforms/android/build.gradle` seems to work around this:

```
android {
    useLibrary 'org.apache.http.legacy'
}
```

Apparently, this was deprecated in API level 22, and removed in API level 23.

**Comment by creat-or** [comment]
I experience the same issue right now.

**Comment by vstirbu** [comment]
does [this](https://github.com/macdonst/cordova-plugin-zeroconf/commit/dad0f3ce6f2794300a904c1eb57b69886754c139) help?

**Comment by vstirbu** [comment]
fixed in 1.2.2
